packages/components/atoms/f-link/src/components/Link.vue

```diff
@@ -114,6 +113,18 @@ export default {
             }
             return 'a';
         }
+    },
+
+    methods: {
+        /**
+         * Set data.linkClass attribute and bind all other $attrs to the component.
+         * linkClass added manually to nested component as inheritAttrs: false does not affect class bindings.
+         */
+        bindAttrs () {
+            const { 'link-class': linkClass, ...atributes } = this.$attrs;
+            this.linkClass = linkClass;
+            return atributes;
```

Comment:
Small typo.
Also, does `$attrs` not have `linkClass` available as a camelCase property?
```suggestion
            const { 'link-class': linkClass, ...attributes } = this.$attrs;
            this.linkClass = linkClass;
            return attributes;
```

Reply:
Also, TIL about using the spread operator in assignment. Maybe calling it something like `otherAttributes` might make its use slightly clearer?

Reply:
No, it is available in `$attrs` exactly as used on the parent element.